src/common.rs

```diff
@@ -47,34 +48,35 @@ pub static MOUNT_SECURE: bool = true;
  * 2. There is no space in the json content, but python version response
  * content contains white space in between keys.
  */
-pub fn json_response_content(
+pub fn set_response_content(
```

Comment:
Because I changed the function's implementation from creating a new response each time to modifying the response body. I think the new name is more appropriate for the function.